### DISCUSSION TITLE
Is it safe to diverge from the setup instructions in these ways?

### DISCUSSION THREAD

**Discussion by dannypernik** [forwardemail/forwardemail.net#232] [Q&A]
Hi, there are a couple ways I'm hoping to diverge from FE's setup instructions for Gmail. Please let me know whether there will be negative consequences for doing the following:

- Check "Use as alias" in Send mail as. Currently, my own email address is added to Cc when I reply all, and I've read that checking this box will fix this issue.
- Change the DNS CNAME record to something other than "fe-bounces". This shows up as a subdomain in the "mailed-by" field of sent emails, and it seems like an odd choice to display on emails that are successfully delivered.

**Comment by dannypernik** [comment]
@titanism I saw your reply to the more recent discussion item - could you provide insight here as well?

**Comment by dannypernik** [comment]
@titanism

**Comment by titanism** [comment]
Hi there - you currently must follow our config/instructions including `fe-bounces` setup otherwise SMTP processing will not work.

**Reply by dannypernik** [reply]
@titanism thank you for the reply. The alias setting in particular is problematic.

- In an email thread, if I'm sending a 2nd consecutive email to a single address (prior to receiving a reply from my first email), my reply is addressed back to myself instead of the intended recipient. It's very easy in this instance to send an email to myself, thinking I've sent it to the recipient, and never see the email again. This can potentially be a big problem, especially now that I have a team of tutors working with me (which is the only reason I started looking for Google Workspace alternatives in the first place).
- The 2nd issue is more of an annoyance and a violation of my OCD sensibilities, but if you click reply all, it will also add my own email address in the To field.
